src/IO.Dyspatch.Tests/IntegrationTest.cs

```diff
@@ -0,0 +1,50 @@
+using System;
+using Xunit;
```

Comment:
personally not a huge fan of xunit, i always preferred nunit but not a big deal

Reply:
Lol I'm just happy I got some sort of unit.